xcdat/axis.py

```diff
@@ -28,6 +29,16 @@
     "Z": {"axis": "Z", "coordinate": "vertical"},
 }
 
+# A dictionary that maps common variable names to coordinate variables. This
+# map is used as fall-back when coordinate variables don't have CF attributes
+# set for ``cf_xarray`` to interpret using `CF_ATTR_MAP`.
+VAR_NAME_MAP: Dict[CFAxisKey, List[str]] = {
+    "X": ["longitude", "lon"],
+    "Y": ["latitude", "lat"],
+    "T": ["time"],
+    "Z": coordinate_criteria["Z"]["standard_name"],
+}
+
```

Comment:
This mapping table addresses the issue here https://github.com/xCDAT/xcdat/pull/343#discussion_r1012275860.

Reply:
I used cf_xarray's `coordinate_criteria["Z"]` because are way too many to list ourselves. Let me know if there is a shorter static list we might want to use instead.

Reply:
https://github.com/xarray-contrib/cf-xarray/blob/614b8ef448112ee51fce60f4db976b6050552dc5/cf_xarray/criteria.py#L40-L54